docs_src/microservices/device/sdk/DeviceSystemEvents.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Device Service SDK - Device System Events
+---
+
+# Device Service SDK - Device System Events
+
+The Device Service SDK offers [APIs](./api/GoDeviceSDK/GoDeviceSDKAPI.md) for publishing device system events, which provide updates on various device processes.
+A System Event DTO is published to the EdgeX MessageBus to provide updates on the status of the device (device discovery progress, profile scan progress, etc.).
+
```

Comment:
Add the statment to describe the SDK will be responsible for publishing progress 0 system event when the device discovery or profile scan process is started and progress 100 or -1 system event when the process is finished.
Also, SDK provides new functions to allow the Device Service implementation to publish other progress with the number of discovered devices.